vault/identity_store_oidc_provider_test.go

```diff
@@ -10,9 +10,587 @@ import (
 	"github.com/hashicorp/vault/helper/namespace"
 	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/stretchr/testify/assert"
 	"gopkg.in/square/go-jose.v2"
 )
 
+func TestOIDC_Path_OIDC_Authorize(t *testing.T) {
+	type args struct {
+		client           *client
+		provider         *provider
+		assignment       *assignment
+		authorizeRequest *logical.Request
+		entityID         string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
```

Comment:
In addition to just `wantErr`, do we also want to test the actual error value to make sure that it contains the errors defined in this package, e.g. `ErrAuthUnsupportedResponseType`?

Reply:
Good idea! Changed to test against actual error codes in 7dc219d.